app/models/solidus_paypal_braintree/gateway.rb

```diff
@@ -375,5 +391,16 @@ def customer_profile_params(payment)
 
       params
     end
+
+    def payment_method_params(payment)
```

Comment:
This method smells like it shouldn't be here, what about a separate class? Anyway I'd call this method `payment_method_attributes_from(payment)` at least or `payment_to_braintree_payment_method_attributes(payment)`, not sure what's best, or worst? 🙂

Reply:
I agree with this, I followed suit with the `customer_profile_params(payment)` method for consistency.